compiler/AST/expr.cpp

```diff
@@ -5858,124 +5885,112 @@ GenRet CallExpr::codegenPrimMove() {
   return ret;
 }
 
-bool CallExpr::codegenPrimMoveRhsIsSpecialPrimop() {
-  CallExpr* rhs    = toCallExpr(get(2));
+static bool codegenIsSpecialPrimitive(BaseAST* target, Expr* e, GenRet& ret) {
   bool      retval = false;
+  CallExpr* ce = toCallExpr(e);
```

Comment:
I suggest renaming 'ce' to 'rhs'. That way a lot of diffs will go away. Both names are equally (non-)suggestive.